### ISSUE TITLE
Liquid Exception: undefined method `transform_values'

### ISSUE THREAD

**Issue by jazibsawar** [robwierzbowski/jekyll-picture-tag#91]
I installed the plugin and configured it as per documentation. But when I tried to use the plugin it gave me this error `Liquid Exception: undefined method 'transform_values' for {nil=>"17.png"}:Hash in about.md`.

My _config.yml:
```
# Picture
picture: 
  source: "assets/images/_fullsize"
  output: "generated" 
  suppress_warnings: false
```

My pitcure.yml:
```
media_presets:
  wide_desktop: 'min-width: 1801px'
  desktop: 'max-width: 1800px'
  wide_tablet: 'max-width: 1200px'
  tablet: 'max-width: 900px'
  mobile: 'max-width: 600px'
markup_presets:
  default:
    markup: auto
    formats: [webp, original]
    widths: [200, 400, 800, 1600]
    media_widths: 
      mobile: [200, 400, 600] 
      tablet: [400, 600, 800]
    size: 800px
    sizes: 
      mobile: 100vw
      desktop: 60vw
    fallback_width: 800
    fallback_format: original
    attributes:
      picture: 'class="awesome" data-volume="11"'
      img: 'class="some-other-class"'
```

**Comment by rbuchberger** [comment]
Thanks for the bug report!

`transform_values` was added to ruby with version 2.4 which was released in 2016. You must be running 2.3 or earlier, which doesn't have that function and causes the error you're showing. That's my fault! The plugin should raise a more helpful error message in that instance. I need to add a ruby version to the gemspec.

To fix your issue, you'll need ruby 2.4 or later. If you're not already using it, [rbenv](https://github.com/rbenv/rbenv) is excellent for handling this.

**Comment by jazibsawar** [comment]
Thanks, @rbuchberger. It solved this issue.
